app/assets/javascripts/profile/authentication.coffee

```diff
@@ -44,7 +44,7 @@ class Identity
   confirmDelete: (ev) ->
     new OX.ConfirmationPopover(
       title: false
-      message: "Are you you want to remove sign in?"
+      message: "Are you you want to remove this authentication method?"
```

Comment:
to match the profile, maybe `"Are you sure you want to remove this sign in option?"` (also note there is a double "you you" in what is in the code now.